tools/serverpod_vscode_extension/package.json

```diff
@@ -8,7 +8,7 @@
     "color": "#020E24",
     "theme": "dark"
   },
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
Question: Shouldn't the version here match the version of Serverpod that we are releasing?

Reply:
wasn't meant to be in this PR, BUT the problem is that I pushed 1.2.0 yesterday and you cannot overwrite a version, you have to increment if you want to update (obvious in hindsight). So unfortunately this means we cannot release the vscode plugin as 1.2.0. 

Not a huge deal I think but still, we should automate this stuff to make sure we are always in sync.